services/scripts/test-e2e.sh

```
@@ -0,0 +1,214 @@
+    cd services
+
+    # Check if system metrics are being written to database
+    docker-compose exec -T alphaloop-database psql -U postgres -d alphaloop_sys -c "
+SELECT COUNT(*) FROM system_metrics WHERE created_at > NOW() - INTERVAL '5 minutes';
+" | grep -v "count" | grep -v "---" | grep -v "rows" | grep -q "[0-9]"
+}
+
+# Performance Tests
+test_response_times() {
+    # Test API response times
+    local start_time=$(date +%s.%N)
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)